plugins/Satellites/src/Satellites.hpp

```diff
@@ -60,8 +60,7 @@ the SGP4/SDP4 algorithms (J.L. Canales' gsat library).
 
 <i>Name and identifiers</i>
 
-Each satellite has a name. It's displayed as a label of the satellite hint and in the list of satellites. Names are not unique though, so they are used only
-for presentation purposes.
+Each satellite has a name. It's displayed as a label of the satellite hint and in the list of satellites. Names are not unique though, so they are used only for presentation purposes.
```

Comment:
Please don't use over-long lines where not really needed (e.g. some table views). See online view on Github. It is easier to review when the lines are fully visible in the box.

Reply:
Strange, see https://github.blog/2013-12-03-soft-wrapping-on-prose-diffs/ (from https://stackoverflow.com/questions/16475439/is-it-possible-to-wrap-long-lines-in-the-github-diff-viewer). If I understand correctly, this should from now on work fine. I'll test this in the next commit on this branch.